x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -838,24 +841,6 @@ describe('when on the endpoint list page', () => {
         expect(emptyState).not.toBe(null);
       });
 
-      it('should display empty state when no log data', async () => {
```

Comment:
This test is now obsolete. The empty state UI is now visible only on API failures as date filters are always "on".